package.json

```diff
@@ -33,17 +33,17 @@
     "browser-sync": "^2.27.8",
     "browser-sync-webpack-plugin": "^2.2.2",
     "concurrently": "^6.0.1",
-    "eslint": "^7.0.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-json": "^2.1.1",
+    "eslint": "^8.11.0",
```

Comment:
Do we want to update the eslint version in the `.pre-commit-config.yaml` to match?